servicetalk-log4j2-mdc-utils/src/main/java/io/servicetalk/log4j2/mdc/utils/ServiceTalkThreadContextMap.java

```diff
@@ -53,8 +54,37 @@ public class ServiceTalkThreadContextMap implements ReadOnlyThreadContextMap, Cl
             " The %s MDC adapters should not be" +
             " loaded at the same time. Please exclude one from your dependencies.%n";
 
+    // FIXME: 0.43 remove the property which should be temporary.
+    private static final String ENABLE_PROPERTY_NAME = "io.servicetalk.log4j2.mdc.utils.capturedContextStorage";
```

Comment:
Consider naming it as an action, like `useCapturedContextStorage` or `disableCapturedContextStorage`.
Might be more convenient to use "disabled" bcz then you can use `Boolean.getBoolean(ENABLE_PROPERTY_NAME)`

Reply:
Let's go with `useCapturedContextStorage`: i just have to change the property name and it does have the 'double-negative' vibes that come from `.disable*=true` properties.